lib/scrabble/player.rb

```diff
@@ -1,21 +1,23 @@
 module Scrabble
 	class Player
-		
+
 		attr_accessor :name, :plays
 
 		def initialize(name)
 			@name = name
 			@plays = []
+			@bag = nil
+			@current_tiles = []
 		end
 
 		def won?
-			if total_score >= 100 
+			if total_score >= 100
 				return true
 			else
 				return false
 			end
 		end
```

Comment:
A possible shortcut here: you can just return `total_score >= 100` since that always evaluates to true or false, without needing to use the `if` statement.

Also, a comment on your `total_score` function below: I would store the total score in an instance variable, rather than re-calculate the score for each play.